electron-main.js

```diff
@@ -114,9 +109,9 @@ function linuxSetup (executable) {
   const os = require('os')
   const { join } = require('path')
   const { spawnSync } = require('child_process')
-  const APP_NAME = 'Keet'
-  const ICON_NAME = 'keet'
-  const DESKTOP_FILE_NAME = 'keet.desktop'
+  const APP_NAME = 'Pear'
+  const ICON_NAME = 'pear'
+  const DESKTOP_FILE_NAME = 'pear.desktop'
```

Comment:
@davidmarkclements Not too sure if this should be changed but having keet here didn't seem right.

Reply:
do we need to gc the keet.desktop file thats added here potentially?

Reply:
Hmm.. My concern would be if something like the keet appimage creates a keet.desktop file, then we don't want to delete that.

Functionally, it should be alright if it exists since we set the handler to pear.desktop anyway but it does leave a Keet entry in the user's application list.

Reply:
would need to do that in `migrate`, @jkcdarunday change sgtm